lib/create-stringify.js

```diff
@@ -19,7 +19,7 @@ function stringify(entity, delims) {
     }
 
     const modObj = entity.mod;
-    const modName = (typeof modObj === 'string' ? modObj : modObj && modObj.name) || entity.modName;
+    const modName = (typeof modObj === 'string' ? modObj : modObj && modObj.name);
```

Comment:
it's incorrect because we support objects with `modName` field here as well as `mod.name`

Looks like we should check instance type of the entity variable before accessing `modObj.name`.